app/game/components/game-input/game-input.js

```diff
@@ -10,10 +10,14 @@ import { verifySolution } from '../../action-creators/round';
 import './game-input.styl';
 
 class GameInput extends Component {
-    render({ result, dispatch, currentRoundIndex }) {
+    render({ result, dispatch, playerInput, currentRoundIndex }) {
         return (
             <div className="game-input">
-                <CodeEditor onChange={bindActionCreators(verifySolution, dispatch)} currentRoundIndex={currentRoundIndex} />
+                <CodeEditor
+                    onChange={bindActionCreators(verifySolution, dispatch)}
```

Comment:
As per [redux docs](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments) `connect` accepts `mapDispatchToProps` as second argument and will automatically bind dispatch to actions creators